README.md

```diff
@@ -2,207 +2,219 @@
+/>
+```
 
-## Usage
 
-`import {`[Calendar](#calendar), [CalendarList](#calendarlist), [Agenda](#agenda)`} from 'react-native-calendars';`
+## Some Code Examples
 
-All parameters for components are optional. By default the month of current local date will be displayed.
+Here are a few code snippets that demonstrate how to use some of the key features of react-native-calendars:  
 
-Event handler callbacks are called with `calendar objects` like this:
 
+### Customize the appearance of the calendar:
 ```javascript
-{
-  day: 1,      // day of month (1-31)
-  month: 1,    // month of year (1-12)
-  year: 2017,  // year
-  timestamp,   // UTC timestamp representing 00:00 AM of this date
-  dateString: '2016-05-13' // date formatted as 'YYYY-MM-DD' string
-}
+<Calendar
+  // Customize the appearance of the calendar
+  style={{
+    borderWidth: 1,
+    borderColor: 'gray',
+    height: 350
+  }}
+  // Specify the current date
+  current={'2012-03-01'}
+  // Callback that gets called when the user selects a day
+  onDayPress={(day) => {console.log('selected day', day)}}
+  // Mark specific dates as marked
+  markedDates={{
+    '2012-03-01': {selected: true, marked: true, selectedColor: 'blue'},
+    '2012-03-02': {marked: true},
+    '2012-03-03': {selected: true, marked: true, selectedColor: 'blue'},
+  }}
+/>
 ```
 
-Parameters that require date types accept `YYYY-MM-DD` formatted `date-strings`, JavaScript date objects, `calendar objects` and `UTC timestamps`.
+### Creating a basic calendar with default settings:
+```javascript
+import React, { useState } from 'react';
+import { Calendar, LocaleConfig } from 'react-native-calendars';
+
+const App = () => {
+  const [selected, setSelected] = useState('');
+
+  return (
+    <Calendar
+      onDayPress={(day) => {
+        setSelected(day.dateString);
+      }}
+      markedDates={{
+        [selected]: { selected: true, disableTouchEvent: true, selectedDotColor: 'orange' },
+      }}
+    />
+  );
+};
 
-Calendars can be localized by adding custom locales to `LocaleConfig` object:
+export default App;
+```
 
+### Configuring the locale:
 ```javascript
-import {LocaleConfig} from 'react-native-calendars';
+import React, { useState } from 'react';
+import { Calendar, LocaleConfig } from 'react-native-calendars';
 
 LocaleConfig.locales['fr'] = {
-  monthNames: [
-    'Janvier',
-    'Février',
-    'Mars',
-    'Avril',
-    'Mai',
-    'Juin',
-    'Juillet',
-    'Août',
-    'Septembre',
-    'Octobre',
-    'Novembre',
-    'Décembre'
-  ],
+  monthNames: ['Janvier', 'Février', 'Mars', 'Avril', 'Mai', 'Juin', 'Juillet', 'Août', 'Septembre', 'Octobre', 'Novembre', 'Décembre'],
   monthNamesShort: ['Janv.', 'Févr.', 'Mars', 'Avril', 'Mai', 'Juin', 'Juil.', 'Août', 'Sept.', 'Oct.', 'Nov.', 'Déc.'],
   dayNames: ['Dimanche', 'Lundi', 'Mardi', 'Mercredi', 'Jeudi', 'Vendredi', 'Samedi'],
   dayNamesShort: ['Dim.', 'Lun.', 'Mar.', 'Mer.', 'Jeu.', 'Ven.', 'Sam.'],
-  today: "Aujourd'hui"
```

Comment:
Why did you remove the "today"? It is necessary for the expandable calendar today button's label

Reply:
Hi @Inbal-Tish - my apologies for the delays. Will work on these asap. I added it back.